components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/openidconnect/dao/RequestObjectDAOImplTest.java

```diff
@@ -1,6 +1,6 @@
 /*
  *
- *   Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *   Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
why this is changed?

Reply:
Addressed with 248b4150dffdfd038ad915e52bc5733554047673